Dockerfile

```diff
@@ -4,9 +4,13 @@ RUN install_clean php-cli php-curl php-xml php-mbstring git unzip
 
 COPY ./ /bug-yield/
 
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer && \
-    cd /bug-yield && composer install && \
-    crontab /bug-yield/docker/crontab
+RUN curl -o /tmp/composer-installer https://getcomposer.org/installer && \
+        curl -o /tmp/composer-installer.sig https://composer.github.io/installer.sig &&  \
+        php -r "if (hash('SHA384', file_get_contents('/tmp/composer-installer')) !== trim(file_get_contents('/tmp/composer-installer.sig'))) { unlink('/tmp/composer-installer'); echo 'Invalid installer' . PHP_EOL; exit(1); }" && \
+        php /tmp/composer-installer --version=2.0.3 --filename=composer --install-dir=/usr/local/bin && \
+        php -r "unlink('/tmp/composer-installer');" && \
+        php -r "unlink('/tmp/composer-installer.sig');" && \
```

Comment:
Can we simplify this with multistage builds using the official Docker image?

By copying from `composer:2` we should be able to get the latest version without running into release 3 at some point. IMO Composer has a good track recording regarding stability.

Reply:
Of course.

Composer has `2.0.4` tags, but I'm all for auto-updating.

Reply:
You could setup @dependabot to also create PR's for updates to the Dockerfile: https://github.com/spejder/msml/blob/master/.github/dependabot.yml#L12-L16

Reply:
@arnested 
Doesn't it do that by default? It's enabled for the repo and creates PRs for composer packages.

Reply:
It's only enabled for composer: https://github.com/reload/bug-yield/blob/master/.github/dependabot.yml

Reply:
Oh, I forgot the config file is in `.github`.. .I'll add docker.